x-pack/plugins/uptime/common/lib/ml.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ML_JOB_ID } from '../constants';
+
+export const getJobPrefix = (monitorId: string) => {
```

Comment:
This code seems fine (and well-commented), but could we please add unit tests for this function? Should only need 2.

Reply:
this actually already has tests in file x-pack/plugins/uptime/public/state/api/__tests__/ml_anomaly.test.ts, this is moved out of public so that it can be used on server side as well.

Reply:
The tests should move with the source file, otherwise things get very confusing. Can you move it as well?

Reply:
have moved the test as well.